spec/system/topic_livestream_authenticated_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+describe "Discourse Livestream - Topic Livestream - Desktop - Authenticated", type: :system, js: true do
```

Comment:
btw, system specs have js enabled by default 🙂 

```suggestion
describe "Discourse Livestream - Topic Livestream - Desktop - Authenticated", type: :system do
```